### ISSUE TITLE
Using master.cf

### ISSUE THREAD

**Issue by 4325123** [tomav/docker-mailserver#1622]
## Looking at using master.cf

* [ ] I would like some feedback concerning a use case

## Description

I'm trying to modify the existing master.cf file so that I can run a script after a message is sent.  I'm no postfix wizard, so it could very well be something I'm doing wrong.  I have a very basic setup of the docker container and have very little customized.  

As a test, I have created and added the following to the postfix-master.cf file...

smtp      inet  n       -       -       -       -       smtpd
   -o content_filter=myhook:dummy
myhook unix - n n - - pipe
  flags=F user=root argv=/tmp/postfix.sh ${sender} ${size} ${recipient}

In the postfix.sh, I have it just touch a file in /tmp to see if it works, but doesn't.  There's also nothing in the maillog that indicates it running, but it may not be logged there.  

I was wondering if there's any other things I need to do to get the docker version to recognize the configs in this file?  Is there any debugging I can see to see if it's running?

Thanks!

**Comment by erik-wramner** [comment]
Honestly I'm not sure, if you can find a tutorial for Postfix it should work here as well. I'll flag it help wanted to see if someone has done something similar.

**Comment by 4325123** [comment]
Thanks for the response.  I think I was close to getting it working and there was an issue with the postfix-master.cf file.  I say this because it didn't seem to be inserting the lines into the postfix configuration, and I was getting this error:

postfix/qmgr[]: warning: connect to transport private/myhook: No such file or directory

When docker would start the instance, I was also getting an error with the postfix-master.cf file. I know it's not much help if I can't provide the message...I'll try to work on getting the error message later on.

I wanted to migrate over to the docker-mailserver but had to get a new OS install finished last night with a working mail server so I just installed it within the OS instead of using the docker image.  I'll probably try to go back to this project in the future, but for now, I had to go back to how things were working before.  I'm still interested to see if the line entered above is still the right syntax, just so that I can get things up and running later with it all.

Thanks!
